test/tests.ts

```diff
@@ -2167,86 +2229,90 @@ describe("passport-saml /", function () {
         });
       });
       it("acme_tools request not signed if missing entry point", function (done) {
-        var samlConfig = {
+        const samlConfig = {
           entryPoint: "",
           issuer: "acme_tools_com",
           callbackUrl: "https://relyingparty/adfs/postResponse",
           privateCert: fs.readFileSync(__dirname + "/static/acme_tools_com.key", "utf-8"),
           authnContext:
             "http://schemas.microsoft.com/ws/2008/06/identity/authenticationmethod/password",
-          signatureAlgorithm: "sha256",
+          signatureAlgorithm: "sha256" as const,
```

Comment:
You shouldn't need `as const` here.

Reply:
we need it, because if we don't add it here, it sets type as string and string is too broad for type 'sha1' | 'sha256‘ | ... used in the module